modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -833,8 +834,51 @@ def raise_exception_if_value_not_present(val, form_obj_description)
         )
       end
 
+      def validate_claim_process_type_bdd!
+        claim_process_type = form_attributes['claimProcessType']
+        if claim_process_type == 'BDD_PROGRAM'
```

Comment:
the claim_type is wrong, it should be "BDD_PROGRAM_CLAIM"
 ```Unmapped VA900 (Backend Response: { status: 415, message: {"errors":[{"id":"3f242539-51fd-4f72-9a1f-301b17502836","status":"415","code":"HttpMessageNotReadable","title":"Unsupported Media Type","detail":"JSON parse error: Cannot deserialize value of type `gov.va.api.lighthouse.benefits.form526.api.v1.ClaimProcessTypeV1` from String \"BDD_PROGRAM\": not one of the values accepted for Enum class: [IDES, FDC_PROGRAM, BDD_PROGRAM_CLAIM, STANDARD_CLAIM_PROCESS]","meta":{"time":"2023-10-...```

Reply:
Discussed with Kayla: Form 526 submissions will have a value of "BDD_PROGRAM" as appropriate for the claimant, but we now map these values to "BDD_PROGRAM_CLAIM" for submission to the EVSS docker container and the PDF Generator.